src/config/settings.py

```diff
@@ -9,17 +9,17 @@
 For the full list of settings and their values, see
 https://docs.djangoproject.com/en/4.2/ref/settings/
 """
-import environ
-
+import os
 from pathlib import Path
 
-
-# Build paths inside the project like this: BASE_DIR / 'subdir'.
-BASE_DIR = Path(__file__).resolve().parent.parent
+import environ
 
 env = environ.Env()
-environ.Env.read_env(env_file=BASE_DIR.parent / ".env")
+env_path = os.path.join(os.path.dirname(__file__), "..", "..", ".env")
```

Comment:
already done in one of previous PRs. 

Remove it from your change list

Reply:
have done my best to minimize it, still imports were wrong ordered so some changes left

Reply:
why are you making changes here? that was changed in different PR, why do we need to revert changes?